roles/st2chatops/meta/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+galaxy_info:
+  description: Install st2chatops
+  author: humblearner
+  company: StackStorm
+  license: Apache
+  min_ansible_version: 2.2
+  platforms:
+    - name: Ubuntu
+      versions:
+        - trusty
+        - xenial
+    - name: EL
+      versions:
+        - 6
+        - 7
+  categories:
+    - st2
+    - devops
+    - chatops
+    - autmation
+dependencies:
+    - role: st2
```

Comment:
Nice! This sounds like a bug discovered.

st2chatops can be configured on separated machine (https://github.com/StackStorm/st2chatops/blob/master/st2chatops.env#L23)  and should not rely on resources created by st2 package.

Can you document the bug in st2chatops repo? It's a packaging issue, we can fix it later.